code/jo.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>JO Gommiswald</title>
+    <link rel="stylesheet" type="text/css" href="style.css"/>
+    <style>
+      @import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap');
+    </style>
+  </head>
+  <body>
+    <nav>
+      <div class="dropdown">
+        <a class="drop white" href="./index.html">NACHWUCHS <i class="arrow down"></i></a>
+        <div class="dropdown-content">
+          <a class="white" href="./jo.html">JO</a><br>
+          <a class="white" href="./piccolo.html">PICCOLO</a>
+          <a class="white" href="./turnen.html">TURNEN</a>
+        </div>
+      </div>
+      <a class="white" href="./termine.html">TERMINE</a>
+      <a class="white" href="./kontakt.html">KONTAKT</a>
+    </nav>
+    <header>
+      <h2>JO</h2>
+    </header>
+    <div class="about">
+      <img src="./images/jo.jpeg">
+      <p>Die JO unterteilt sich in folgende Gruppen:<br><br>
+        Ski Fungruppe: Teilnahme an den Trainings-, sowie am LinthCUP- und Clubrennen<br><br>
+        Ski Funracegruppe: Teilnahme an den Trainings, an allen LinthCUP- und Clubrennen<br><br>
+        Ski Renngruppe: Teilnahme an den Trainings, Lizenz- oder Animationsrennen, inklusive LinthCUP- und Clubrennen,
+        Teilnahme am Trainingslager im Herbst und an den Trainingsweekends erwünscht<br><br>
+        Snowboard: Für Kinder, die selbständig Snowboardfahren können und weiter an ihrer Fahrtechnik arbeiten möchten</p>
+    </div>
+    <div class="coach">
+      <p><img src="./images/urs.jpeg" style="width:80px;height:80px;">
+      Ansprechsperson:<br><a class="black" href="./kontakt.html">Urs Kaufmann</a></p>
```

Comment:
```suggestion
      <p>
        <img src="./images/urs.jpeg" style="width:80px;height:80px;">
        Ansprechsperson:
        <br>
        <a class="black" href="./kontakt.html">
          Urs Kaufmann
        </a>
      </p>
```
You could improve the readability here by changing line for each tag (with proper indentation).